awx/main/utils/common.py

```diff
@@ -581,12 +582,8 @@ def get_system_task_capacity():
     from django.conf import settings
     if hasattr(settings, 'SYSTEM_TASK_CAPACITY'):
         return settings.SYSTEM_TASK_CAPACITY
-    try:
-        out = subprocess.check_output(['free', '-m'])
-    except subprocess.CalledProcessError:
-        logger.exception('Problem obtaining capacity from system.')
-        return 0
-    total_mem_value = out.split()[7]
+    mem = psutil.virtual_memory()
+    total_mem_value = mem.total/1024/1024
```

Comment:
if we're taking the 7th output of 'free -m', this should be mem.available, not mem.total

Reply:
Oops, never mind.